packages/@aws-cdk/aws-redshift/lib/parameter-group.ts

```diff
@@ -62,17 +63,46 @@ export class ClusterParameterGroup extends ClusterParameterGroupBase {
    */
   public readonly clusterParameterGroupName: string;
 
+  /**
+   * The parameters in the parameter group
+  */
+  readonly parameters: { [name: string]: string };
+
+  /**
+   * The underlying CfnClusterParameterGroup
+  */
+  private readonly resource: CfnClusterParameterGroup;
+
   constructor(scope: Construct, id: string, props: ClusterParameterGroupProps) {
     super(scope, id);
-
-    const resource = new CfnClusterParameterGroup(this, 'Resource', {
+    this.parameters = props.parameters;
+    this.resource = new CfnClusterParameterGroup(this, 'Resource', {
       description: props.description || 'Cluster parameter group for family redshift-1.0',
       parameterGroupFamily: 'redshift-1.0',
-      parameters: Object.entries(props.parameters).map(([name, value]) => {
-        return { parameterName: name, parameterValue: value };
-      }),
+      parameters: this.parseParameters(),
     });
 
-    this.clusterParameterGroupName = resource.ref;
+    this.clusterParameterGroupName = this.resource.ref;
+  }
+  private parseParameters(): any {
+    return Object.entries(this.parameters).map(([name, value]) => {
+      return { parameterName: name, parameterValue: value };
+    });
+  }
+
+  /**
+   * Adds a parameter to the parameter group
+   *
+   * @param name the parameter name
+   * @param value the parameter name
+   */
+  public addParameter(name: string, value: string): void {
```

Comment:
Same question as on addToParameterGroup. If there's a context where users may want to add several, can we let them do it all in one shot instead of calling this over and over again?